### DISCUSSION TITLE
Use STK for search

### DISCUSSION THREAD

**Discussion by mmalloush** [trilinos/Trilinos#12606] [General]
Hi,
This is a small question here. Is it possible to do the following with stk coarse search:
I have a bounding box, and I need to know which of my mesh nodes lie n the box
I have an stk mesh which I populated from an exodus file

**Comment by alanw0** [comment]
Yes you could do that.
I see this was closed as resolved. If you want more discussion or more detail, let us know.

**Reply by mmalloush** [reply]
Thanks. Yes this was resolved. I am using stk::search::coarse_search to do that, after I define a vector of pairs of all my mesh nodes paired with their global identities. Then the coarse_search function provided me with a resultant vector of pairs from which I get the global identities of all the nodes that lie in the region of interest.
